### PR TITLE
Generic Styled Templates

### PR DESCRIPTION
**Pull request by WendyBeth**
@mattsears @JesseHerrick 

This PR overlays styling from Pond onto the admin panel and uses a custom wrapper from simple_form to define the admin forms. It adds two routes - `/dashboard` and `/styleguide`.

Next steps:
+ Create "super" controller that AdminController inherits from, so developer can choose between basic CRUD functionality or to write their own controller by inheriting from a different, empty controller
+ Add a method of hooking into an app's auth system, so that the bare-bones of auth are in place (admin must be signed in to see/use CEO admin panel)
+ Add basic javascript functionality such as datepicker, image preview, dropdown, flash messages, etc. (port in directives from other projects)
+ Add svg files & make inline_svg gem work
+ Comment & document thoroughly
+ Maybe auto-generate links in the header? Or generate _header view in project so developer can customize it. Same w/ dashboard.

### CODE REVIEW

**Comment by JesseHerrick** [comment]
@WendyBeth I glanced at this but I have to leave for an exam. I'll take a deeper look in a few hours and let you know what I think. Thanks for doing this!

**Comment by WendyBeth** [comment]
Thanks! I plan on changing the directory names, such as under assets, to /admin from /ceo, in order to follow how you've done it. The next thing I'm working on is an easier way to override the default CRUD controller, and since that also adds a few more directories, I've made the change to the directory names in the branch I'm working on that in.

One other note: The guard/minitest-guard gems are commented out b/c they broke the build on older Ruby versions and I couldn't figure out a way to remove them/use an older version of them depending on the Ruby version of the environment.

File: ceo.gemspec [hunk 1]
```diff
@@ -18,12 +18,17 @@ Gem::Specification.new do |s|
...
   s.add_dependency 'rails', '>= 4'
+  s.add_dependency 'autoprefixer-rails'
+  s.add_dependency 'sassc-rails'
+  s.add_dependency 'inline_svg'
+  s.add_dependency 'simple_form'
 
   s.add_development_dependency 'capybara'
-  s.add_development_dependency 'guard'
-  s.add_development_dependency 'guard-minitest'
+  # s.add_development_dependency 'guard'
```
Comment (JesseHerrick) [inline ceo.gemspec]: You can just delete these.

File: lib/ceo/rails/routes.rb [hunk 2]
```diff
@@ -19,6 +19,8 @@ def admin_for(*rsrcs, &block)
       options[:concerns] = :pageable
 
       namespace :admin do
+        get 'dashboard', to: 'admin#dashboard'
```
Comment (JesseHerrick) [inline lib/ceo/rails/routes.rb]: There has to be a better way to define this route other than recreating it every time the method is called. Any thoughts @WendyBeth @mattsears?

Comment (WendyBeth) [inline lib/ceo/rails/routes.rb]: You're completely right @JesseHerrick, thanks for pointing that out. I'll add that to my to-do list.

**Pull request merged**